.circleci/config.yml

```diff
@@ -1133,9 +1133,9 @@ workflows:
           context:
             - oplabs-gcr
       - docker-publish:
-          name: chain-mon-docker-publish
+          name: wd-mon-docker-publish
           docker_file: ./ops/docker/Dockerfile.packages
-          docker_name: chain-mon
+          docker_name: wd-mon
```

Comment:
Should match this name: https://github.com/ethereum-optimism/optimism/blob/21b7cf01b3e5f4efb66de8ec857df10c59ed6f41/ops/docker/Dockerfile.packages#L128